app/helpers/application_helper/toolbar/drift_view.rb

```diff
@@ -11,7 +11,8 @@ class ApplicationHelper::Toolbar::DriftView < ApplicationHelper::Toolbar::Basic
       'fa fa-bars fa-rotate-90 fa-lg',
       N_('Compressed View'),
       nil,
-      :url => "drift_compress"),
+      :url   => "drift_compress",
```

Comment:
@romanblanco couldn't find `drift_compress` in miq_product_features. So should it stay like this?